BraveRewardsUI/Tipping/TippingViewController.swift

```diff
@@ -5,14 +5,20 @@
 import UIKit
 import BraveRewards
 
+enum BravePublisherSocialMedia: String {
```

Comment:
We should use same naming as desktop for that which seems to be `MediaType` or `PublisherMediaType`, cc @kylehickinson for awareness

On desktop I see stuff like `#define YOUTUBE_MEDIA_TYPE "youtube"` in `static_values.h`